src/wolff_client.erl

```diff
@@ -103,6 +103,15 @@ check_connectivity(Hosts, ConnConfig) ->
             {error, tr_reasons(Reasons)}
     end.
 
+-spec check_connectivity([host()], kpro:conn_config(), topic()) -> ok | {error, any()}.
+check_connectivity(Hosts, ConnConfig, Topic) ->
+  case get_metadata(Hosts, ConnConfig, Topic, []) of
+    {ok, _} ->
+      ok;
+    {error, Errors} ->
```

Comment:
Should check if `[] = Errors` (no errors / no hosts checked)?

```suggestion
    {error, []} ->
      ok;
    {error, Errors} ->
```

Reply:
Ack